### DISCUSSION TITLE
Route all traffic vpn by NetBird peer

### DISCUSSION THREAD

**Discussion by riokmlee** [netbirdio/netbird#795] [Ideas & Feature Requests]
Is it possible to allow netbird endpoint to selectively route all traffice through any connected peer/network?
Just like a normal vpn function, using some oversea address to do some regional specific work ;p
Maybe admin can assign/restrict only some peer can act as "router".
Most likely this feature will not auto enable by default and need to select peer and enable manually.

**Comment by tbleiker** [comment]
I would love to see this as well. See #723.

**Comment by ser** [comment]
it's really strange it's not available

**Comment by mlsmaycon** [comment]
Hello, folks; thanks for the request. The issue #723 is the best tracker for this feature. 

As mentioned, NetBird doesn't support routing all traffic just yet, but it supports routing network ranges. See https://docs.netbird.io/how-to/routing-traffic-to-private-networks for more details.

**Reply by werlitong** [reply]
In this direction, maybe in a future, could we route traffic to a specific domain (like google.com) to exit through a peer/node?
